README.rst

```diff
@@ -112,7 +293,7 @@ The `designation` is the first line and conveys several important details about
 
 *     extra format data e.g. ``<hxsurface>`` specifying that an AmiraMesh file will contain HyperSurface data
 
-A series of `definitions` follow that refer to data found in the data pointer sections that either begin with the word ‘define’ or have ‘n’ prepended to a variable. For example:
+        A series of `declarations` defines the sturcture and layout of the data and how many elements the corresponding array, vector or matrix contains. A declaration lline starts with the word â€˜defineâ€™ or in files created by early Amira versions have â€˜nâ€™ prepended. For example:
```

Comment:
Is `declarations` the official term?

Reply:
Ok checked in the reference guide for version 5. they talk about arrays and entries

> [...] one-dimensional array Nodes containing entries of type float[3] called
Coordinates, as well as a one-dimensional array Tetrahedra containing entries of
type int[4] called Nodes. [...]

and further down they describe the statement defining the array 

> [...] The statement define Nodes 4 defines a one-dimensional array of size 4.
Later on, this array can be referenced using the name Nodes. Similarly, a state-
ment define Array 100 100 defines a two-dimensional array of size 100 x
100 [...]

and the statement defining the data element / item
> The statement Nodes { float[3] Coordinates } = @1 specifies that
for each element of the array Nodes defined earlier, three floating point numbers
(floats) should be stored. These data are given the name Coordinates and a tag
in this line and will appear below as a tagged block with the marker @1

So their wording uses `array` and `element` definition and `tag`, `tagged block` and marker when they mean `@N` symbol. So if i would change wording to `array definitions` and `element definitions` if we want strictly stick to their wording. Should we also reword the corresponding grammar symbols and variable names to consistent and ease reading and understanding code for future contributors and ourselves when having to amend in one of our next lives?